scanner.yml

```diff
@@ -48,11 +43,19 @@
     DOCKER_DRIVER: overlay2
     DOCKER_TLS_CERTDIR: ""
 
+.boost_rules:
```

Comment:
^ the part about listening on TCP without auth to Docker daemon, I see you say you plan on remove, let's just make sure we have JIRA task to track and not forget

Reply:
all good, been explained